components/frontend/src/widgets/toast.test.jsx

```diff
@@ -6,10 +6,9 @@ import { showConnectionMessage, showMessage } from "./toast"
 
 vi.mock("react-toastify")
 
-beforeEach(() => {
-    vi.resetAllMocks()
-    history.push("")
-})
+beforeEach(() => history.push(""))
+
+afterEach(() => vi.resetAllMocks())
```

Comment:
What is reset about the mocks here, is it their call content? Funny that doesn't automatically happen after each test case

Reply:
This one has been removed, see previous comment.